src/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
This file (probably as a ts not tsx file) will be the entrypoint of the apps using the component library thus this file should contain exports of the modules you want to expose to the world.

Reply:
Yes, we will do this when there are modules to export.

Reply:
:+1: